packages/@react-spectrum/menu/stories/ActionMenu.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import {action} from '@storybook/addon-actions';
+import {ActionMenu} from '../src/ActionMenu';
+import {Item} from '../';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+storiesOf('ActionMenu', module)
```

Comment:
For new stories files, please use Storbyooks CSF syntax, there are a few examples in the repo
https://github.com/adobe/react-spectrum/blob/main/packages/@react-spectrum/accordion/chromatic/Accordion.chromatic.tsx
sorry, we haven't updated our templating to include that

Reply:
done